src/java/org/apache/cassandra/index/sai/disk/v1/postings/PostingListRangeIterator.java

```diff
@@ -67,7 +67,7 @@ public class PostingListRangeIterator extends KeyRangeIterator
     private final long rowIdOffset;
 
     private boolean needsSkipping = false;
-    private PrimaryKey skipToToken = null;
+    private PrimaryKey skipToKey = null;
```

Comment:
Yeah, naming holdover from a past where this was a token?

Reply:
I thought I'd just clean that up while I was there. It was confusing.